go.mod

```diff
@@ -2,9 +2,11 @@ module github.com/crossplane/addon-oam-kubernetes-remote
 
 go 1.13
 
+replace github.com/crossplane/crossplane-runtime => github.com/hasheddan/crossplane-runtime v0.0.0-20200320144815-bb562fce2536
```

Comment:
Staying in draft until https://github.com/crossplane/crossplane-runtime/pull/133 is merged